distribution/docs/src/main/resources/_contents/_draft-catalog-contents/managing-catalog-contents.adoc

```diff
@@ -0,0 +1,47 @@
+
+The ${ddf-catalog} provides a framework for storing, searching, processing, and transforming information.
+
+Clients typically perform query, create, read, update, and delete (QCRUD) operations against the Catalog.
```

Comment:
I've never seen "query" and "read" listed separately before. This is normally just "CRUD".